PROTOCOL.md

```diff
@@ -274,6 +276,7 @@ Field Name | Data Type | Description
 -|-|-
 appId | String | A unique identifier for the application performing the transaction
 version | Long | An application-specific numeric identifier for this transaction
+lastUpdated | Long | Optional. The time this transaction was introduced, as milliseconds since the epoch
```

Comment:
lastUpdated | Option[Long] | The time when this transaction action is created, in milliseconds since the Unix epoch.